src/main/java/mate/academy/Main.java

```diff
@@ -1,55 +1,56 @@
-        firstCinemaHall.setCapacity(100);
-        firstCinemaHall.setDescription("first hall with capacity 100");
-
-        CinemaHall secondCinemaHall = new CinemaHall();
-        secondCinemaHall.setCapacity(200);
-        secondCinemaHall.setDescription("second hall with capacity 200");
-
-        CinemaHallService cinemaHallService = null;
-        cinemaHallService.add(firstCinemaHall);
-        cinemaHallService.add(secondCinemaHall);
-
-        System.out.println(cinemaHallService.getAll());
-        System.out.println(cinemaHallService.get(firstCinemaHall.getId()));
-
-        MovieSession tomorrowMovieSession = new MovieSession();
-        tomorrowMovieSession.setCinemaHall(firstCinemaHall);
-        tomorrowMovieSession.setMovie(fastAndFurious);
-        tomorrowMovieSession.setShowTime(LocalDateTime.now().plusDays(1L));
-
-        MovieSession yesterdayMovieSession = new MovieSession();
-        yesterdayMovieSession.setCinemaHall(firstCinemaHall);
-        yesterdayMovieSession.setMovie(fastAndFurious);
-        yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
-
-        MovieSessionService movieSessionService = null;
-        movieSessionService.add(tomorrowMovieSession);
-        movieSessionService.add(yesterdayMovieSession);
-
-        System.out.println(movieSessionService.get(yesterdayMovieSession.getId()));
-        System.out.println(movieSessionService.findAvailableSessions(
-                        fastAndFurious.getId(), LocalDate.now()));
+    private static final Injector injector = Injector.getInstance("mate.academy");
+
+    public static void main(String[] args) throws RegistrationException, AuthenticationException {
+
+        // 0. INJECTING
+        final var userService
+                = (UserService) injector.getInstance(UserService.class);
+        final var authenticationService
+                = (AuthenticationService) injector.getInstance(AuthenticationService.class);
+
+        // 1. CREATING
+        // User
+        User user1 = new User();
+        user1.setEmail("user1@gmail.com");
+        user1.setPassword("password");
+
+        User user2 = new User();
+        user2.setEmail("user2@gmail.com");
+        user2.setPassword("password");
+
+        User user3 = new User();
+        user3.setEmail("user3@gmail.com");
+        user3.setPassword("password");
+
+        // 2. ADDING
+        // User
+        userService.add(user1);
+        userService.add(user2);
+        userService.add(user3);
+
+        // 3. GETTING BY EMAIL
+        // User
+        System.out.println("\n *** CREATE AND READ FUNCTIONS TEST *** \n");
+        System.out.println(userService.findByEmail("user1@gmail.com"));
+        System.out.println(userService.findByEmail("user2@gmail.com"));
+        System.out.println(userService.findByEmail("user3@gmail.com"));
+
+        // 4. REGISTRATION
+        // Correct
+        authenticationService.registerUser("jan@gmail.com", "haslojana");
```

Comment:
The `registerUser` method is called, but it's unclear if it returns a value or handles exceptions properly. Ensure that it aligns with the method's expected behavior in the `AuthenticationService` class.